bundle/android/strings.xml

```diff
@@ -394,6 +393,7 @@ translators: %s: Select control option value e.g: "Auto, 25%". -->
     <string name="gutenberg_native_video_caption_empty" tools:ignore="UnusedResources">Video caption. Empty</string>
     <!-- translators: accessibility text. %s: video caption. -->
     <string name="gutenberg_native_video_caption_s" tools:ignore="UnusedResources">Video caption. %s</string>
+    <string name="gutenberg_native_waiting_for_connection" tools:ignore="UnusedResources">Waiting for connection</string>
```

Comment:
Part of the Media Upload improvements https://github.com/WordPress/gutenberg/pull/56937/files#diff-7293d2894b8d052856d29cabd7e46010cc719d11ed6fbe2652199174733d106fR163